nestkernel/nestmodule.cpp

```diff
@@ -723,6 +723,10 @@ NestModule::ResetKernelFunction::execute( SLIInterpreter* i ) const
    Name: ResetNetwork - Reset the dynamic state of the network.
    Synopsis: ResetNetwork -> -
    Description:
+
+   ResetNetwork is deprecated as of NEST 2.18 and will be removed in NEST3,
```

Comment:
There should probably be a space between `NEST` and `3` for consistency.

Reply:
Now changed to NEST 3.0 throughout.